slurm/0_Introductory/BasicExample.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -l
+
+##Select the appropriate cluster from the README.md placeholders
+#SBATCH --cluster=[cluster]
+
+##Select the appropriate partition from the placeholders							
+#SBATCH --partition=[partition]
+
+##qos is generally same as [partition]
```

Comment:
```suggestion
##qos is usually the same as [partition] - refer to [CCR docs](https://docs.ccr.buffalo.edu/en/latest/hpc/jobs/#slurm-directives-partitions-qos)
```
NOTE: those docs are getting updated during Feb downtime with more info on QOS